src/components/Explorer/DropdownContents.vue

```diff
@@ -30,15 +32,30 @@ export default defineComponent({
 </script>
 
 <style lang="scss" module>
+:global {
+  .fade-enter-active,
+  .fade-leave-active {
+    transition: opacity 0.5s;
+  }
+  .fade-enter,
+  .fade-leave-to {
+    opacity: 0;
+  }
+  .fade-enter-to,
+  .fade-leave {
+    opacity: 1;
+  }
```

Comment:
```suggestion
  .fade-enter-from,
  .fade-leave-to {
    opacity: 0;
  }
  .fade-enter-to,
  .fade-leave-from {
    opacity: 1;
  }
```
クラス名の指定ミスですね
ただこのドキュメントのほうが読みやすいのでこっちでお願いします:pray:
https://v3.vuejs.org/guide/transitions-enterleave.html#transitioning-single-elements-components

Reply:
vue 3でtransitionはAPIが変更されたのでその影響ですね